docs/vt-extensions/color-palette-update-notifications.md

```diff
@@ -43,3 +43,13 @@ e.g. by configuration.
 
 Please have a look at our example C++ [source code](https://github.com/contour-terminal/contour/blob/master/examples/detect-dark-light-mode.cpp)
 in order to see how to implement this in your own application.
+
+## Adoption State
+| Support  | Terminal/Toolkit/App | Notes                                                                   |
+|----------|----------------------|-------------------------------------------------------------------------|
+| ✅       | Contour              |                                                                         |
+| ✅       | Ghostty              | since `1.0.0`                                                           |
+| ✅       | Kitty                | since `0.36.3`                                                          |
+| not yet  | tmux                 | see tracker: [tmux#4269](https://github.com/tmux/tmux/issues/4269)      |
+| not yet  | WezTerm              | see tracker: [wezterm#6454](https://github.com/wez/wezterm/issues/6454) |
```

Comment:
With little research, I'm actually really happy that Neovim adopted it as well: https://github.com/neovim/neovim/pull/31350

Reply:
Linked in that issue I also found an issue for zellij, which I have also added.